safe_control_gym/controllers/mpc/gp_mpc.py

```diff
@@ -152,6 +152,7 @@ def __init__(
             constraint_tol=constraint_tol,
             # runner args
             # shared/base args
+            use_gpu=use_gpu,
```

Comment:
GPMPC parent class is MPC whose parent is BaseController
But MPC's constructor never calls super's constructor.
Is this correct?

Reply:
Damn you're right my bad, should I remove this or make MPC call the BaseController? I feel like every controller should call the BaseController super since it has some logic rather than repeating the logic, which will be more important if we ever decide to add more logic to BaseController as well

Reply:
I think MPC should call the BaseController, although on the short run now (as @adamhall designed it as a base class itself) it might just bloat the object.

Reply:
wait so the conclusion is to add the call to BaseController to MPC even tho it bloats it a bit right?

Reply:
yes with CC to @adamhall so that he can work on removing potentially duplicated code